src/main/java/ch/wisv/areafiftylan/model/Ticket.java

```diff
@@ -63,11 +64,27 @@ public Ticket() {
         //JPA Only
     }
 
+    public void setUpForTransfer(User u){
+        this.setLockedForTransfer(true);
+        this.setTransferGoalOwner(u);
+    }
+
+    public void finalizeTransfer(){
```

Comment:
I'd move this logic to the service, as this is supposed to be just the model class